File: VB/WindowsApplication1/MyGaugeControl.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Text
Imports DevExpress.XtraReports.UI
Imports DevExpress.XtraGauges.Win

Namespace WindowsApplication1
	Public Class MyGaugeControl
		Inherits WinControlContainer
		Private gauge As New GaugeUserControl()

		Public Sub New()
			MyBase.New()
			Me.WinControl = gauge
			Me.DrawMethod = WinControlDrawMethod.UseWMPaintRecursive

		End Sub

	End Class
End Namespace

File: CS/WindowsApplication1/MyGaugeControl.cs
using System;
using System.Collections.Generic;
using System.Text;
using DevExpress.XtraReports.UI;
using DevExpress.XtraGauges.Win;

namespace WindowsApplication1 {
    public class MyGaugeControl: WinControlContainer {
        GaugeUserControl gauge = new GaugeUserControl();

        public MyGaugeControl() : base() {
            this.WinControl = gauge;
            this.DrawMethod = WinControlDrawMethod.UseWMPaintRecursive;
        }
    }
}
